src/security-council-mgmt/SecurityCouncilManager.sol

```diff
@@ -347,15 +393,39 @@ contract SecurityCouncilManager is
                     if (nomineeGovernor.isContender(proposalId, newAddress)) {
                         revert NewMemberIsContender(proposalId, newAddress);
                     }
+                    if (nomineeGovernor.isNominee(proposalId, newAddress)) {
+                        revert NewMemberIsNominee(proposalId, newAddress);
+                    }
                 }
             }
         }
 
         lastRotated[newAddress] = block.timestamp;
+        rotatedTo[msg.sender] = newAddress;
+        rotationNonce[msg.sender] = currentRotationNonce + 1;
         Cohort cohort = _swapMembers(msg.sender, newAddress);
         emit MemberRotated({replacedAddress: msg.sender, newAddress: newAddress, cohort: cohort});
     }
 
+    /// @inheritdoc ISecurityCouncilManager
+    function setRotatingTo(address newMemberAddress, bytes calldata signature) external {
+        uint256 currentRotationNonce = rotationNonce[msg.sender];
+        // we enforce that a the new address is an eoa in the same way do
+        // in NomineeGovernor.addContender by requiring a signature
+        bytes32 digest = getSetRotatingToHash(msg.sender, currentRotationNonce);
+        address newAddress = ECDSAUpgradeable.recover(digest, signature);
```

Comment:
nit: we can inline the digest to make it syntactically similar to the one in `rotateMember`